nilearn/regions/tests/test_rena_clustering.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from nose.tools import assert_equal, assert_not_equal, assert_raises
+from sklearn.externals.joblib import Memory
```

Comment:
```suggestion
try:
    from sklearn.externals.joblib import Memory
except ImportError:
    from joblib import Memory
```
as joblib is unvendored from scikit-learn 0.12